cmd/otk-gen-partition-table/main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+
+	"github.com/osbuild/images/internal/buildconfig"
+	"github.com/osbuild/images/internal/cmdutil"
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/internal/otkdisk"
+	"github.com/osbuild/images/pkg/blueprint"
+	"github.com/osbuild/images/pkg/disk"
+	"github.com/osbuild/images/pkg/osbuild"
+)
+
+type Input struct {
```

Comment:
Since this basically becomes our public API, I think that all of these have to be documented with docstrings.